src/reporters/http_sender.js

```diff
@@ -45,7 +45,7 @@ export default class HTTPSender {
     this._url = URL.parse(options.endpoint);
     this._username = options.username;
     this._password = options.password;
-    this._timeoutMS = options.timeoutMS || DEFAULT_TIMEOUT_MS;
+    this._timeoutMs = options.timeoutMs || DEFAULT_TIMEOUT_MS;
```

Comment:
let's keep this backwards compatible: `options.timeoutMs || options.timeoutMS || DEFAULT_TIMEOUT_MS;`

Reply:
Sure, can do. `timeoutMS` was not referenced elsewhere in this codebase, so I felt safe changing it. Is it possible that it's used by code outside the codebase that passes `timeoutMS`?

Reply:
http sender is not hidden under some `internal` package, so technically it's a public API, and I'd prefer to not break anyone who may be wiring components manually.

Reply:
Thanks for the info. Will update as requested.